pkg/reconciler/tenancy/initialization/apibinder/clusterworkspacetype.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: tenancy.kcp.dev/v1alpha1
+kind: ClusterWorkspaceType
+metadata:
+  name: apibinder
+spec:
+  initializer: true
+  defaultChildWorkspaceType: Universal
+  allowedChildWorkspaceTypes:
+    - Any
```

Comment:
(no action required) maybe `*` instead of `Any`?

Reply:
The use of `Any` predates this PR. `*` does not match our cluster name validating regexes, and it's nice to use the same type here to get the validation ... /shrug